brdnet/download_categorized_data.ipynb

```diff
@@ -838,146 +903,144 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 20,
    "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
```

Comment:
I would also include a suppressMessages flag here for notebook readability. e.g.
suppressMessages(lapply(included.study.list, 
       function(x) tryCatch(download_study(x, type = "rse-gene", 
                                           outdir = file.path(recount.dir, x)),
                            error= function(e) NULL)))